Aztec/Classes/Public/Extensions/CGRect+Helpers.swift

```diff
@@ -0,0 +1,56 @@
+import CoreGraphics
+
+extension CGRect {
+    /// The position to which a rect is anchored when resized vertically.
+    ///
+    enum VerticalAnchor {
+        case top
+        case middle
+        case bottom
+    }
+
+    /// The position to which a rect is anchored when resized horizontally.
+    ///
+    enum HorizontalAnchor {
+        case left
+        case center
+        case right
+    }
+
+    /// Return a rect resized to a specific size, while anchoring it to one of
+    /// their edges or centers.
+    ///
+    /// By default, changing a CGRect size will resize it anchored to its top
```

Comment:
In other words, I think the description should explain what the method does, and not why we particularly needed it in the first place.